Sprint-3/alarmclock/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Title here</title>
+    <title>Alarm clock app</title>
   </head>
   <body>
     <div class="centre">
```

Comment:
If you have more time, could try to make the app look nicer using some css. overall good work and functionality.

Reply:
Thank you! I've updated the CSS file and made the app look visually nicer. 🌸